lms/djangoapps/learner_recommendations/utils.py

```diff
@@ -55,6 +55,67 @@ def _remove_user_enrolled_course_keys(user, course_keys):
     return enrollable_course_keys
 
 
+def _has_country_restrictions(product, user_country):
+    """
+    Helper method that tell whether courses or programs have any country restrictions.
```

Comment:
Lets add details of the implementation.
```suggestion
    Helper method that tell whether the product (course or program) has any country restrictions. A product is restricted for the user if the country in which user is logged in from:
    - is in the "block list" or
    - is not in the "allow list" if the "allow list" is not empty. If it is empty, then all locations can access it.
```

Reply:
I think we should add details about the function's purpose and how to use this function, not about technical implementation.
What do you think about that?

Reply:
This is an explanation on what True and False means.